kano_apps/AppManage.py

```diff
@@ -118,15 +118,15 @@ def download_app(app_id_or_slug):
     icon_path = '/tmp/{}.{}'.format(app_id_or_slug, icon_file_type)
     rv, err = download_url(data['icon_url'], icon_path)
     if not rv:
-        msg = _("Unable to download the application ({})").format(err)
+        msg = _(u"Unable to download the application ({})").format(err)
         raise AppDownloadError(msg)
 
     # Check if the app is going to run well on the hardware
     if 'min_performance_score' in data and \
        has_min_performance(data['min_performance_score']):
 
         msg = _(
-            "{} won't be downloaded "
+            u"{} won't be downloaded "
```

Comment:
Yeah it will be because it's decoded from JSON. Thats why the string needs to be unicode as well.